feature/login/src/commonMain/kotlin/com/b1nd/dodam/login/navigation/LoginNavigation.kt

```diff
@@ -7,9 +7,9 @@ import androidx.navigation.NavGraphBuilder
 import androidx.navigation.NavOptions
 import androidx.navigation.compose.composable
 import androidx.navigation.navOptions
-import login.LoginScreen
+import com.b1nd.dodam.login.LoginScreen
 
-const val LOGIN_ROUTE = "login"
+const val LOGIN_ROUTE = "com/b1nd/dodam/login"
```

Comment:
루트에는 패키지 네임이 들어가지 않아도 됩니다.
```suggestion
const val LOGIN_ROUTE = "login"
```